ocaml/parsing/jane_syntax.ml

```diff
@@ -1767,3 +1767,25 @@ module Extension_constructor = struct
       (* See Note [Outer attributes at end] *)
       { ext_ctor with pext_attributes = ext_ctor.pext_attributes @ attrs }
 end
+
+module Dummy_arguments = struct
+  type expression = Dummy_argument
+
+  let embedded_name = "jane.non_erasable.dummy_arguments"
```

Comment:
I would love to use `Embedded_name.to_string` but it's not exposed. Its comment says "not expose" which I suppose has a reason behind it.

Reply:
Could we define this constant in `jane_syntax_parsing.ml`? That way we don't have to expose `Embedded_name.to_string` and don't duplicate its implementation (kind of).

Reply:
My impression of `Jane_syntax_parsing` is that it's not specific to any extension or syntax - so probably shouldn't have a `string` specific to dummy arguments? Maybe it's not dangerous to expose `Embedded_name.to_string`?

Reply:
I don't have a strong preference between either approach.